api.swagger.yaml

```diff
@@ -14928,7 +14929,7 @@ components:
           type: string
           example: 'More information here.'
     list-list-content-schema:
-      allOf:
```

Comment:
This should be allOf it is combining the common schema with the specific object. anyOf would mean it can be either or both.